frontend/app/work_packages/services/work-package-field-service.js

```diff
@@ -254,6 +263,8 @@ module.exports = function(
         break;
       case 'User':
         displayStrategy = 'user';
```

Comment:
@hirtie-maxim there is a `break` missing here, leading to a problem where all user fields will be treated as `daterange` inputs.